### ISSUE TITLE
Adding voids to profile outside of constructor throws error

### ISSUE THREAD

**Issue by secretlyagoblin** [hypar-io/Elements#221]
**Describe the bug**
The pattern

```
var circle = Polygon.Circle(10);
var innerCircle = Polygon.Circle(9).Reversed();
var profile = new Profile(circle);
profile.Voids.Add(innerCircle);
```
throws an error as the voids list has not been instantiated.

Expected behavior is for this to just work as voids by default is a list of 0 elements.

Workaround of `profile.Voids = new List<Polygon>(){innerCircle};` is fine for now but might trip up a newcomer and isn't the nicest.

**Comment by secretlyagoblin** [comment]
I built the elements repo and disabled the existing validator checks and the 3d pipes are working well enough in gltf, so that'll tide me over for now!

![image](https://user-images.githubusercontent.com/13952064/70954319-98adc480-20d1-11ea-89d5-b346326bc879.png)

**Comment by ikeough** [comment]
Closed in #295.
